src/deepsparse/loggers/prometheus_logger.py

```diff
@@ -168,19 +174,20 @@ def get_prometheus_metric(
         description_template.format(
             metric_name=metric._type, identifier=identifier, category=category
         ),
+        ["model_name"],
         registry=registry,
     )
 
 
-def format_identifier(identifier: str) -> str:
+def format_identifier(identifier: str, namespace: str = _NAMESPACE) -> str:
     """
     Replace forbidden characters with `__` so that the identifier
     digested by prometheus adheres to
     https://prometheus.io/docs/concepts/data_model/#metric-names-and-labels
     :param identifier: The identifier to be formatted
     :return: The formatted identifier
     """
-    return re.sub(r"[^a-zA-Z0-9_]", "__", identifier).lower()
+    return f"{namespace}_{re.sub(r'[^a-zA-Z0-9_]', '__', identifier).lower()}"
```

Comment:
nice to leave a comment describing what a regex pattern does (or is intended to do :) )

Reply:
I think that the docstrings are quite explicit no? When the user opens the link, they can instantaneously see the same regex there: `[a-zA-Z_:][a-zA-Z0-9_:]*` (actually colon is not really allowed)